app/Actions/Diagnostics/Checks/PHPVersionCheck.php

```diff
@@ -12,15 +12,28 @@ public function check(array &$errors): void
 		// I hereby solemnly  declare this code as covered !
 		// @codeCoverageIgnoreStart
 
+		// 30 Nov 2019	 => 7.2 = DEPRECATED = ERROR
+		// 28 Nov 2019	 => 7.4 = RELEASED   => 7.3 = WARNING
+		// 26 Nov 2020	 => 8.0 = RELEASED   => 7.4 = WARNING
+		//? 6 Dec 2020	 => 7.3 = DEPRECATED = ERROR
 		$php_error = 7.4;
 		$php_warning = 7.4;
-		$php_latest = 7.4;
+		$php_latest = 8;
 
-		//! 30 Nov 2019	 => 7.2 = DEPRECATED = ERROR
-		//! 28 Nov 2019	 => 7.4 = RELEASED   => 7.3 = WARNING
-		//! 26 Nov 2020   => 8.0 = RELEASED   => 7.4 = WARNING
-		//! 6 Dec 2020	 => 7.3 = DEPRECATED = ERROR
-		// 28 Nov 2021	 => 7.4 = DEPRECATED = ERROR
+		//! 28 Nov 2021	 => 7.4 = DEPRECATED = ERROR
+		// $php_error = 8;
+		// $php_warning = 7.4;
```

Comment:
```suggestion
		// $php_warning = 8;
```
It seems odd to have a warning <7.4 but error <8.

Reply:
True, but then you get two lines which basically say the same thing. :)

Reply:
I refer you to https://github.com/LycheeOrg/Lychee/pull/972/files#diff-d27eeec1f1336ba359adca471272fecf5ae6dc987c69cdb375aae7e6e11d15e9R33-R36

Reply:
Also, with the comments you have, you'd have an `Error: Upgrade to PHP 7.4 or higher` warning.

Reply:
oohhh oupsie.